spec/features/visitor_signs_up_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+require 'rails_helper'
+
+feature 'Visitor signs up' do
+  scenario 'with valid email and password' do
+    sign_up_with 'valid@example.com', 'password'
+
+    expect(page).to have_content('Logout')
+  end
+
+  scenario 'with invalid email' do
+    sign_up_with 'invalid_email', 'password'
+
+    expect(page).to have_content('Sign up')
+  end
+
+  scenario 'with blank password' do
+    sign_up_with 'valid@example.com', ''
+
+    expect(page).to have_content('Sign up')
+  end
```

Comment:
The problem is that sign up is based on Devise and I don't know what kind of flash message should I expect.
What's more, when I change code to "expect(flash[:notice]).to be_present" it appears a failure 
```
 NameError:
       undefined local variable or method `flash' for #<RSpec::ExampleGroups::VisitorSignsUp:0x00000007e04f00>"
```